### ISSUE TITLE
Histogram/Distribution metrics do not work

### ISSUE THREAD

**Issue by jbaldassari** [GoogleCloudPlatform/opentelemetry-operations-js#473]
Please answer these questions before submitting a bug report.

### What version of OpenTelemetry are you using?

1.8.0

### What version of Node are you using?

18.4.0

### What did you do?

Just define a `Histogram` instrument with OpenTelemetry and record some values:

```
import opentelemetry, {DiagConsoleLogger, DiagLogLevel, Histogram, diag} from '@opentelemetry/api';
import {
  ExplicitBucketHistogramAggregation,
  MeterProvider,
  PeriodicExportingMetricReader,
  View,
} from '@opentelemetry/sdk-metrics';
import {MetricExporter} from '@google-cloud/opentelemetry-cloud-monitoring-exporter';

diag.setLogger(new DiagConsoleLogger(), DiagLogLevel.ALL);

const instrumentName = 'poc/opentelemetry/response/latency';

const responseTimeView = new View({
  aggregation: new ExplicitBucketHistogramAggregation([0, 25, 50, 75, 100, 125, 150, 175, 200], true),
  instrumentName,
});

const meterProvider = new MeterProvider({
  views: [responseTimeView],
});

const exporter = new MetricExporter({
  prefix: 'custom.googleapis.com',
  projectId: 'your-project-id-here',
});

meterProvider.addMetricReader(
  new PeriodicExportingMetricReader({
    exportIntervalMillis: 5_500,
    exporter,
  }),
);

opentelemetry.metrics.setGlobalMeterProvider(meterProvider);

const meter = meterProvider.getMeter('opencensus-poc');

const latencyMillisHistogram = meter.createHistogram(instrumentName, {
  unit: 'ms',
});

latencyMillisHistogram.record(sample);
```

### What did you expect to see?

No errors from `PeriodicExportingMetricReader`

### What did you see instead?

```
Send TimeSeries failed: One or more TimeSeries could not be written: Request was missing field timeSeries[0].points[0].doubleValue.; Request was missing field timeSeries[1].points[0].doubleValue.
{"stack":"Error: PeriodicExportingMetricReader: metrics export failed (error Error: Send TimeSeries failed: One or more TimeSeries could not be written: Request was missing field timeSeries[0].points[0].doubleValue.; Request was missing field timeSeries[1].points[0].doubleValue.)\n    at PeriodicExportingMetricReader._runOnce (/home/james/dev/git/mabl/perf-test-sandbox/node_modules/@opentelemetry/sdk-metrics/build/src/export/PeriodicExportingMetricReader.js:56:19)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async Timeout._onTimeout (/home/james/dev/git/mabl/perf-test-sandbox/node_modules/@opentelemetry/sdk-metrics/build/src/export/PeriodicExportingMetricReader.js:63:17)","message":"PeriodicExportingMetricReader: metrics export failed (error Error: Send TimeSeries failed: One or more TimeSeries could not be written: Request was missing field timeSeries[0].points[0].doubleValue.; Request was missing field timeSeries[1].points[0].doubleValue.)","name":"Error"}
```

### Additional context

The problem is that the `transformValueType` function from `transform.ts` is not correctly mapping the value type for a distribution metric.  It's sending a `ValueType` of `DOUBLE` when it should be `DISTRIBUTION`:

```
sent metric descriptor {
  type: 'custom.googleapis.com/poc/opentelemetry/response/latency',
  description: '',
  displayName: 'poc/opentelemetry/response/latency',
  metricKind: 'CUMULATIVE',
  valueType: 'DOUBLE',
  unit: 'ms',
  labels: []
}
```

I'll have a PR up shortly with a fix.

**Comment by justin--** [comment]
Has this update gone in? The current version 0.15.0 still has this problem.

**Comment by jbaldassari** [comment]
@justin-- the fix (https://github.com/GoogleCloudPlatform/opentelemetry-operations-js/pull/474) was merged, but I don't think a release has been published that includes it yet.  `0.15.0` was the version I was using when I discovered it.  To get around the issue I built a new version from source and deployed it to an internal npm repo, but it would be great if we could get an official release out that includes the fix.
